git/git.go

```diff
@@ -27,6 +29,25 @@ func Pull(repo, repoDir string) error {
 	return nil
 }
 
+func GetBranchHeadSha(repo, branch string) (string, error) {
+	repoUrl := "git@github.com:" + repo + ".git"
+	w := log.WithField("id", "git-ls-remote").WriterLevel(log.DebugLevel)
+	defer w.Close()
+	stdout := new(bytes.Buffer)
+	mw := io.MultiWriter(stdout, w)
+	cmd := command.New(command.WithStdout(mw), command.WithStderr(w))
```

Comment:
I don't see any reason to log the output of ls-remote so I think this is unnecessary. You can remove the multiwriter

Reply:
Updated. 👍